app/controllers/discovered_hosts_controller.rb

```diff
@@ -262,4 +293,17 @@ def skip_bullet
   ensure
     Bullet.enable = true if defined? Bullet
   end
+
+  def get_interfaces
+    @host.interfaces.each do |interface|
+      @interfaces << {:identifier => interface["identifier"], :type => interface["type"], :mac => interface["mac"], :ip => interface["ip"]? interface["ip"] : "N/A", :primary => interface["primary"], :provision => interface["provision"]}
+    end
+  end
+
+  def add_custom_facts
+    if @host.subnet
+      discovery_subnet = "#{@host.subnet.name + " (" + @host.subnet.network + ")"}"
```

Comment:
Weird usage of embedding :-) I'd probably go with `"#{host.subnet.name} (#{host.subnet.network})"`.